spec/request/csrf_request_spec.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+# This file is for exercising routes that should require CSRF protection.
+# It is very much a WIP
+
+# TODO: \
+# Lighthouse API endpoints
+# Check routes.rb for other `POST` routes
+
+require 'rails_helper'
+
+RSpec.describe 'CSRF scenarios', type: :request do
+  around do |example|
+    original_val = ActionController::Base.allow_forgery_protection
+    ActionController::Base.allow_forgery_protection = true
+    example.run
+    ActionController::Base.allow_forgery_protection = original_val
+  end
```

Comment:
Is this exists because `ActionController::Base.allow_forgery_protection = false` in test mode? 

1. Probably worthy of a comment.
2. Any merit to doing a before(:all) and after(:all) to set these instead? I'm not sure if setting this value incurs any performance hit.

Reply:
Yes.
1. 👍 
2. Agreed, seems like a good use case for `before/all :all`